test/integration/test_dog.py

```diff
@@ -0,0 +1,499 @@
+import math
```

Comment:
i'd add this somewhere in https://github.com/kornia/kornia-benchmark

we are trying to reduce the size of the repository to keep as light as possible, including heavy tests like this.

in case we want to include as part of our ci, ideally it must be only torch. I see that big part of the opencv functionality you use is found already here in kornia.

/cc @ducha-aiki @johnnv1

Reply:
I also don't see why we would have this test here. If not in kornia-benchmark maybe add it as part of the tutorials?  Also the images could be included in the kornia/data repository

Reply:
I shortened it to absolute minimum - to 100 lines. It uses torch/kornia only except for a) cv.imread/cvtColor and b)  np.rot90 to prepare imgs for logic based on cv (in another file, another 100 lines), which gives better results (reprojection error ~0.001 vs ~0.1), I suspect because of stability of kornia.geometry.RANSAC vs. cv.findHomography.
 
Maybe it is still heavy-weight - I can move it to kornia-benchmark, I just wanted to show what this change does (ensures/comes very close to rotational equivariance of DoG) - I guess in kornia-benchmark it will only be possible to see the improvement once this code is released.

Reply:
@vicsyl there should be no `cv2` in the main repo. How about the following:
1) put the smallest possible image size into `torch.save` file, similarly how we test LoFTR here: https://github.com/kornia/kornia/blob/master/test/feature/test_loftr.py#L37

The rest of benchmark can go to the kornia-benchmark and there is no restrictions on the imports there.